index.html

```diff
@@ -0,0 +1,301 @@
+        </section>
+        <section class="about bg-white">
+          <h2 class="title">About Myself</h2>
+          <p class="details">
+            I am a software developer! If you have a project you need coded, do not hestiate to
+            contact me.
+          </p>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET US CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="#"
+                  ><img src="image/tw.png" alt="Twitter"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="image/linkedin.png"
+                    alt="LinkedIn"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/medium.png" alt="Medium"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/github.png" alt="Github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/v.png" alt="V"
+                /></a>
+              </li>
+            </ul>
+            <button class="btn-default" type="button">Get my bio-data</button>
+          </div>
+          <ul class="skill-list">
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Languages</h3>
+                <button class="btn-chev" type="button">
+                  <img src="image/chev-dow.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+              <ul class="language-list">
+                <li>
+                  <img src="image/javascript.png" alt="JavaScript" />JavaScript</li>
+                <li>
+                  <img src="image/html.png" alt="HTML" />HTML</li>
+                <li>
+                  <img src="image/css.png" alt="CSS" />CSS</li>
+              </ul>
+            </li>
+
+            <!-- /languages -->
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Frameworks</h3>
+                <button class="btn-chev" type="button">
+                  <img src="image/chev-right.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+            </li>
+            <!-- /Frameworks -->
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Skills</h3>
+                <button class="btn-chev" type="button">
+                  <img src="image/chev-right.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+            </li>
+            <!-- /Skills -->
+          </ul>
+        </section>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, please fix the about me page and correctly position the languages, skills, frameworks section to correctly match the Figma design .that will make your project more professional. you got this 👍 

Figma design:
![image](https://user-images.githubusercontent.com/90524466/186486013-fb2cfd0b-651e-490b-a90c-e654dadaa0aa.png)


Your design:
![image](https://user-images.githubusercontent.com/90524466/186486107-42267cbc-c358-4f1c-aec7-eb2bc5d5cf0e.png)